docs/compatibility.md

```diff
@@ -82,6 +82,6 @@ The following apps are supported by govuk-docker to some extent.
    - ✅ transition
    - ✅ travel-advice-publisher
    - ⚠ whitehall
-      * Who knows, really - several tests are failing, lots pass ;-)
+      * Most tests pass, but several tests fail consistently due to issues with govuk-docker
```

Comment:
```suggestion
      * Most tests pass, but several tests fail consistently and we haven't investigated why yet
```

Reply:
That's also true, but I was trying here to give direction should anyone decide to investigate.

At least when debugging the test failures for govuk-guix, I found several tests had quite specific requirements on the environment, so my guess is that something is up with the Dockerfile that's in use.